lib/ansible/modules/source_control/github_organization.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/python
+#
+# (c) 2017, Peter Murray <peter.murray@osirisoft.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: github_organization
+short_description: Create/Rename a GitHub organization.
+description:
+    - Create or Rename a GitHub organization.
+version_added: "2.4"
+options:
+  state:
+    description:
+      -  Whether to create or rename an organization
+    choices: ['present', 'rename']
```

Comment:
Is it possible to remove an organization?

Reply:
No, the API does not support removal. I have raise a feature request on this with GitHub, if enough people request the feature, it will be implemented, but there is no API to cover this in 2.11, only create and rename as per https://developer.github.com/v3/enterprise-admin/orgs/